src/FredData.jl

```diff
@@ -12,7 +12,7 @@ import JSON
 
 export
        # Fred object
-       Fred, get_api_url, set_api_url!, get_api_key,
+       Fred, fred_key, get_api_url, set_api_url!, get_api_key,
```

Comment:
`load_fred_key` (see below) will not need to be exported as it is not part of the public API. Instead, any user who requires to use the method can import it explicitly.